Javascript/Question-two.js

```diff
@@ -0,0 +1,11 @@
+const form = document.getElementById("form__list");
+
+form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  const elements = event.target.elements.namedItem('query2');
+
+  const isCorrect = elements[1].checked && elements[2].checked && !elements[0].checked;
```

Comment:
mustbe isCorrect is True if only the correct answer is selected

Reply:
hello, if it were for one answer I would have to make the change you ask for, but it is valid that two answers are true and one is false.